src/mmda/types/span.py

```diff
@@ -24,32 +24,32 @@ def to_json(self) -> Dict:
 
     @classmethod
     def from_json(cls, span_dict) -> "Span":
-        box_dict = span_dict.get('box')
+        box_dict = span_dict.get("box")
         if box_dict:
-            box = Box.from_json(box_dict=span_dict['box'])
+            box = Box.from_json(box_dict=span_dict["box"])
         else:
             box = None
-        return Span(start=span_dict['start'], end=span_dict['end'], box=box)
+        return Span(start=span_dict["start"], end=span_dict["end"], box=box)
 
-    def __lt__(self, other: 'Span'):
+    def __lt__(self, other: "Span"):
         if self.id and other.id:
             return self.id < other.id
         else:
             return self.start < other.start
 
     @classmethod
-    def small_spans_to_big_span(cls, spans: List['Span']) -> 'Span':
+    def small_spans_to_big_span(cls, spans: List["Span"]) -> "Span":
         # TODO: add warning for unsorted spans or not-contiguous spans
         # TODO: what happens when Boxes cant be merged?
-        start = None
-        end = None
-        for span in spans:
+        start = spans[0].start
+        end = spans[0].end
```

Comment:
was this broken?

Reply:
ah yes, this actually was broken. surprised it hadnt come up before. a minimal test case under the old code:

```
span = Span(0, 2)
Span.small_spans_to_big_span([span])
```
results in
```
TypeError: '<' not supported between instances of 'int' and 'NoneType'
```

whereas with the fix, you get:
```
Span(start=0, end=2, box=None)
```